src/WCSimConstructMaterials.cc

```diff
@@ -1039,7 +1041,8 @@ void WCSimDetectorConstruction::ConstructMaterials()
 
    G4double PP[NUM] = { 1.4E-9*GeV,6.2E-9*GeV};
    G4double RINDEX_blacksheet[NUM] =
-     { 1.6, 1.6 };
```

Comment:
Thank you for seeing it. I think it is a bug from a very old study I did 3 years ago, trying to increase blacksheet reflection by +10% hardcodedly, instead of using mac file. I have modified it back to 1.6, than you very much for seeing it. The difference was found to be negligible for E61 (so even more for HK), so I do not think we should re-check results.